<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsWidget</class>
 <widget class="QWidget" name="GeneralSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Behaviour</string>
     </property>
     <layout class="QGridLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="pauseOnStart">
        <property name="text">
         <string>Pause On Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="confirmPowerOff">
        <property name="text">
         <string>Confirm Power Off</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="saveStateOnExit">
        <property name="text">
         <string>Save State On Exit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="loadMemoryCardsFromSaveStates">
        <property name="text">
         <string>Load Memory Cards From Save States</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="startFullscreen">
        <property name="text">
         <string>Start Fullscreen</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="renderToMain">
        <property name="text">
         <string>Render To Main Window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Emulation Speed</string>
     </property>
     <layout class="QGridLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="emulationSpeed">
          <property name="minimum">
           <number>25</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>25</number>
          </property>
          <property name="pageStep">
           <number>25</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="emulationSpeedLabel">
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="enableSpeedLimiter">
        <property name="text">
         <string>Enable Speed Limiter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="increaseTimerResolution">
        <property name="text">
         <string>Increase Timer Resolution</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>On-Screen Display</string>
     </property>
     <layout class="QGridLayout" name="formLayout_5">
      <item row="2" column="0">
       <widget class="QCheckBox" name="showOSDMessages">
        <property name="text">
         <string>Show Messages</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="showFPS">
        <property name="text">
         <string>Show FPS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="showSpeed">
        <property name="text">
         <string>Show Emulation Speed</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="showVPS">
        <property name="text">
         <string>Show VPS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
